2021/results/aws_join.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+PYTHON=python3
+
+FORCE_STATUS_COMPUTATION=0
+Cloud="cloud"
+Parallel="parallel"
+
+smtlib_root=$1
+
+for track in Cloud Parallel; do
+    echo -n "$track: "
+    InstanceMap="../prep/selection/final/${!track}-map.csv"
+    InstanceStatuses="${!track}-track/aws-instance-statuses.csv"
+
+    if [ "$FORCE_STATUS_COMPUTATION" == "1" -o \! -e "$InstanceMap" ]; then
+
+        if [ $# != "1" ]; then
+            echo "Instance statuses not available.  Please uncompress"\
+                 "them to $InstanceStatuses or provide the path to smtlib-base"\
+                 "as argument."
+            echo "Usage: $0 <path-to-smtlib>"
+            exit 1
+        fi
+        $PYTHON ./aws-extract-starexec-status.py -m $InstanceMap -p $smtlib_root \
+            > ${InstanceStatuses}
+        cd ${!track}-track
+        tar zcf aws-instance-statuses.tar.gz aws-instance-statuses.csv
+        cd ..
+    fi
+
+    echo ${!track}-track/results.csv
+
+    echo "Removing duplicate instances and fixing cvc5-gg PARSE_ERRORs to timeouts"
+    for name in $(csvcut -c 'smtlib name' \
+            ../prep/selection/final/${!track}-map.csv \
+            |sort |uniq -d); do
+        csvgrep -c 'smtlib name' -m "$name" \
```

Comment:
The selection files contain duplicates because the criterion for selecting `hard` and `unsolved` instances is not mutually exclusive.  There are 5 duplicates in the cloud track and 4 duplicates in the parallel track.